src/components/DevDummyLogin/index.js

```diff
@@ -156,9 +156,11 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
 
           const userId = await JwtDecode(response.token).user_id;
           const anonymousUserId = await JwtDecode(response.anonymousToken).user_id;
-          setProfileId({
+          setAuth({
             anonProfileId: anonymousUserId,
-            signedProfileId: userId
+            signedProfileId: userId,
+            token: response.token,
+            anonymousToken: response.anonymousToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setAuth` function is now receiving more parameters (`token`, `anonymousToken`) than the previous `setProfileId` function. Ensure that these additional parameters are handled correctly within the `setAuth` function and that they don't introduce any unexpected behavior or security issues. Also, make sure that the `token` and `anonymousToken` are stored securely and not exposed to potential security risks.
```diff
- setProfileId({
-    anonProfileId: anonymousUserId,
-    signedProfileId: userId
- });
+ setAuth({
+    anonProfileId: anonymousUserId,
+    signedProfileId: userId,
+    token: response.token,
+    anonymousToken: response.anonymousToken
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->